### PR TITLE
Spider's Den Server Widget

### PR DESCRIPTION
**Pull request by Ghost-3**
Demo:

### CODE REVIEW

File: src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+     */
+    @Override
+    public void updateContent() {
+        this.addSimpleIcoText(Ico.MAP, "Area:", Formatting.DARK_AQUA, 41);
+        this.addSimpleIcoText(Ico.NTAG, "Server ID:", Formatting.GRAY, 42);
+        this.addSimpleIcoText(Ico.EMERALD, "Gems:", Formatting.GREEN, 43);
+
+        BroodmotherState broodmotherState = parseTab();
+        this.addSimpleIcoText(
+                Ico.SPIDER_EYE, "Broodmother: ", broodmotherState.formatting(), broodmotherState.text());
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java:84]: Have this all on the same line

File: src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ */
+public class SpidersDenServerWidget extends Widget {
+
+    private static final MutableText TITLE =
+            Text.literal("Server Info").formatted(Formatting.DARK_AQUA, Formatting.BOLD);
+
+    /**
+     * Broodmother Mini-Boss tab states
+     */
+    private enum BroodmotherState {
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java:22]: This is missing the "Imminent" state
image

**Review by AzureAaron — CHANGES_REQUESTED** [review]

File: src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java (hunk 2 above)
Comment (AzureAaron) [reply src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java:22]: It's also missing the "Slain" state
image

**Comment by Ghost-3** [comment]
All fixed

**Review by AzureAaron — APPROVED** [review]

**Pull request merged**